src/components/MessageInput.js

```diff
@@ -128,7 +128,12 @@ class MessageInput extends PureComponent {
       props.editing,
       props.initialValue,
     );
-    this.state = { ...state, sending: false };
```

Comment:
@DanC5 can we add some code comments as to explain whats happening? Just reading through code, its a bit hard to understand

Reply:
@vishalnarkhede I added comments for the various steps in the flow, let me know if you have questions!